src/scss/_common.scss

```diff
@@ -113,6 +113,10 @@
         }
     }
 
+    .mermaid svg[id*='mermaid'] {
```

Comment:
should this change definitely go to transform and not to mermaid extension, for example?

Reply:
the problem is that transform package overrides these styles in it's packages/transform/src/scss/_common.scss style sheet